app/controllers/publishers/vacancies/base_controller.rb

```diff
@@ -3,13 +3,11 @@
 class Publishers::Vacancies::BaseController < Publishers::BaseController
   include Publishers::Wizardable
 
-  helper_method :step_process, :vacancy
+  private
 
-  def step_process
-    # TODO: We currently have to do this kinda thing in a lot of places thanks to `wicked`
-    #       Find a better way!
-    current_step = defined?(step) ? step : :review
+  helper_method :current_step, :step_process, :vacancy
 
+  def step_process
     ::Publishers::Vacancies::VacancyStepProcess.new(
```

Comment:
Do we need this `::`?

Reply:
Probably not, but I assumed they'd put it there for a reason.

Reply:
I'll try PRing it out and see if anyone complains.